aws/resource_aws_macie2_classification_job_test.go

```diff
@@ -0,0 +1,307 @@
+package aws
+
+import (
+	"fmt"
+	"os"
+	"testing"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/macie2"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/terraform"
+	"github.com/terraform-providers/terraform-provider-aws/aws/internal/naming"
+)
+
```

Comment:
This should include a test `TestAccAwsMacie2ClassificationJob_basic` that checks the return values when only the required parameters are passed.